LivingCity/traffic/b18TrafficLaneMap.cpp

```diff
@@ -109,7 +112,7 @@ void B18TrafficLaneMap::createLaneMapSP(const std::shared_ptr<abm::Graph>& graph
     const int numWidthNeeded = ceil(edgesData[tNumMapWidth].length / kMaxMapWidthM);
     edgesData[tNumMapWidth].numLines = numLanes;
     edgesData[tNumMapWidth].nextInters = std::get<1>(std::get<0>(x));
-    edgesData[tNumMapWidth].nextInters = graph_->vertex_map_[std::get<1>(std::get<0>(x))];
+    edgesData[tNumMapWidth].nextInters = std::get<1>(std::get<0>(x));
```

Comment:
It seems to be that after replacing the code with the new ID numbering, we ended up with two equal assignments. I think we can remove one of them.